pallets/exchange/src/tests.rs

```diff
@@ -177,6 +177,10 @@ fn sell_test_pool_finalization_states() {
 				user_3_sell_intention_id,
 			)
 			.into(),
+			orml_tokens::Event::Reserved(asset_a, 2, 1000000000000).into(),
+			orml_tokens::Event::Reserved(asset_b, 3, 2000000000000).into(),
+			orml_tokens::Event::Reserved(asset_b, 2, 2000000000).into(),
+			orml_tokens::Event::Reserved(asset_b, 3, 4000000000).into(),
```

Comment:
was wondering if this repetitive part could be setup and reused as a function?

Reply:
yes.possibly. but i have noticed little discrepancy in this, so i investigate that first. And when all good, we can handle this kind of improvement.